### PR TITLE
[IDX-4537] multiple pixels template

### PR DESCRIPTION
**Pull request by marat-kh**

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -14,9 +14,8 @@ ___INFO___
   "version": 1,
   "securityGroups": [],
   "displayName": "IDX",
-  "categories": ["ADVERTISING", "TAG_MANAGEMENT"],
   "brand": {
-    "id": "brand_idx",
```
Comment (ffqs) [inline template.tpl]: revert

File: template.tpl [hunk 2]
```diff
@@ -14,9 +14,8 @@ ___INFO___
   "version": 1,
   "securityGroups": [],
   "displayName": "IDX",
-  "categories": ["ADVERTISING", "TAG_MANAGEMENT"],
```
Comment (ffqs) [inline template.tpl]: revert

File: template.tpl (hunk 1 above)
Comment (marat-kh) [reply template.tpl]: fixed

File: template.tpl (hunk 2 above)
Comment (marat-kh) [reply template.tpl]: fixed

**Review by ffqs — APPROVED** [review]

**Pull request merged**